README.adoc

```diff
@@ -8,6 +8,11 @@ This program is an onboard media control for the University of Calgary's Solar C
 :imagesdir: assets/images
 image::plan.jpg[Plan]
 
+== Programs Needed to Run Program 
```

Comment:
Doesn't quite roll off the tongue. Since this is all for getting images, just call it `Acquiring Album Art`.
Also note to the user that they only need to run it once.